sklearn/externals/funcsigs.py

```diff
@@ -20,8 +20,8 @@
 __all__ = ['BoundArguments', 'Parameter', 'Signature', 'signature']
 
 
-_WrapperDescriptor = type(type.__call__)
```

Comment:
Looking at the commit history it looks like testing-cabal/funcsigs is a continuation of aliles/funcsigs that stopped in December 2013. testing-cabal/funcsigs is also the repo listed in PyPI: https://pypi.python.org/pypi/funcsigs.